chain/client/src/stateless_validation/chunk_validator/mod.rs

```diff
@@ -225,10 +225,16 @@ pub(crate) fn send_chunk_endorsement_to_block_producers(
     let endorsement = ChunkEndorsement::new(chunk_header.chunk_hash(), signer);
     for block_producer in block_producers {
         if signer.validator_id() == &block_producer {
-            // Unwrap here as we always expect our own endorsements to be valid
-            chunk_endorsement_tracker
+            // Our own endorsements are not always valid (see issue #11750).
+            if let Err(err) = chunk_endorsement_tracker
                 .process_chunk_endorsement(chunk_header, endorsement.clone())
-                .unwrap();
+            {
+                tracing::warn!(
```

Comment:
this should be error unless there is an expected reason?

Reply:
An `ERROR` log is printed just before inside `process_chunk_endorsement`, this one is almost duplicate, but it could be on error level as well

Reply:
ah ok saw it. thanks.